thunder/core/prims.py

```diff
@@ -2676,23 +2678,51 @@ def _uniform_meta(
 )
 
 
+def _get_and_update_rng_state_meta(
+    seed: IntegerProxy | int | None, offset: IntegerProxy | int | None, device: devices.Device
+) -> tuple[IntegerProxy, IntegerProxy]:
+    seed_is_none = seed is None
+    offset_is_none = offset is None
+    utils.check(
+        not (seed_is_none ^ offset_is_none),
+        lambda: f"seed and offset must be given in pair (they are both None only used on first use of get_and_update_rng_statebut), but got {seed}, {offset}",
+    )
+    if not seed_is_none:
+        utils.check_type(seed, (IntegerProxy, int))
+    if not offset_is_none:
+        utils.check_type(offset, (IntegerProxy, int))
+    utils.check(
+        device.devicetype is devices.DeviceType.CUDA,
+        lambda: f"get_and_update_rng_state is supported for CUDA only",
+        exception_type=NotImplementedError,
+    )
+    return numberproxy(int, 0), numberproxy(int, 0)
```

Comment:
@jjsjann123 Maybe in the future these should be number proxies without values?

Reply:
Can we use `None` instead of `0` already in this PR?

Reply:
I tried using None, and it works in this PR, but it has some problem in backward